README.md

```diff
@@ -1,69 +1,39 @@
 # training-specific-template
 
 This repository is intended to serve as a template for creating a repository for an individual Data Lab workshop.
-The repository structure and use of GitHub pages is intended to gather all material required to administer a workshop in one user-friendly place.
+The repository structure and use of GitHub Pages is intended to gather all material required to administer a workshop in one user-friendly place.
 We use a template repository approach for maintainability.
 
 ## Creating a repository for an individual training workshop
 
-First, you will need to create a new repository.
+First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+This will create a new repository with the same layout as this template repository under the user or organization you choose as the owner of the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should be the owner.
 
-### Data Lab workshops
+If you are using GitHub from a paid tier account (e.g., GitHub Pro, GitHub Team, or GitHub Enterprise), you can set the visibility of this repository as private; the associated GitHub Pages website will still be public.
+However, if you are using GitHub from a _free_ account (e.g., GitHub Free for individuals or organizations), you must set the visibility of this repository as public in order for GitHub Pages to be visible, as described in this [documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).
```

Comment:
Slight wording change to remove double visible/visibility. And a slightly bigger link target.
```suggestion
However, if you are using GitHub from a _free_ account (e.g., GitHub Free for individuals or organizations), you must set the visibility of this repository as public in order for GitHub Pages to be active, as described in [this documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).
```